js/src/forum/components/SettingsPage.js

```diff
@@ -60,6 +61,14 @@ export default class SettingsPage extends UserPage {
       })
     );
 
+    items.add('tokens',
```

Comment:
I feel like this section should go alongside email/password/sso settings, not 100% sure though.

Reply:
Still WIP. we can change that later